federation/cmd/federation-apiserver/app/server.go

```diff
@@ -221,8 +232,11 @@ func Run(s *options.ServerRunOptions) error {
 	installBatchAPIs(m, genericConfig.RESTOptionsGetter)
 	installAutoscalingAPIs(m, genericConfig.RESTOptionsGetter)
 
-	sharedInformers.Start(wait.NeverStop)
-	return m.PrepareRun().Run(wait.NeverStop)
+	err = m.PrepareRun().NonBlockingRun(stopCh)
+	if err == nil {
+		sharedInformers.Start(stopCh)
```

Comment:
@sttts Is there any danger in starting the informers after the server can accept connections?